src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -692,33 +863,39 @@ private void addTaskUDFValue(Task task, Row row)
                break;
             }
 
+            case BOOLEAN:
+            {
+               value = STATICTYPE_UDF_MAP.get(row.getString("udf_text"));
```

Comment:
Hmm... perhaps. I guess all that's needed to suite my use case but leave the default alone is to add a `setMapIndicatorUdfToFlag` sort of function to the reader classes to toggle the behavior, but I wonder if there's something more flexible that makes sense and isn't a large code lift. I'll have to think about this one a little bit more.